packages/expo-cli/src/commands/utils/npm.ts

```diff
@@ -101,10 +113,30 @@ export async function downloadAndExtractNpmModuleAsync(
   npmName: string,
   props: ExtractProps
 ): Promise<void> {
-  const url = await getNpmUrlAsync(npmName);
-
-  Log.debug('Fetch from URL:', url);
-  await extractNpmTarballFromUrlAsync(url, props);
+  Log.debug(`Looking for tarball for ${npmName} in ${getCacheFilePath()}...`);
+  try {
+    const cachePath = getCacheFilePath();
+    // Perform dry-run to get actual filename for resolved version
+    const filename = (((await npmPackAsync(
```

Comment:
My understanding is that this command is used `Create a tarball from a package` I don't see how this is analogous to the fetch command.